rest/config.go

```diff
@@ -331,9 +331,76 @@ func (dbConfig *DbConfig) setup(name string) error {
 		dbConfig.Server = &urlStr
 	}
 
+	// If the sync function is referenced from an external file or http(s) endpoint
+	// unlike the inline JavaScript definition, read it from the specified path.
+	if dbConfig.Sync != nil {
+		sync, err := loadJavaScript(*dbConfig.Sync)
+		if err != nil {
+			return err
+		}
+		dbConfig.Sync = base.StringPtr(sync)
```

Comment:
`base.StringPtr` is a convinence function to get the pointer for a literal string, if we have a variable already, we don't need it.

```suggestion
		dbConfig.Sync = &sync
```

Reply:
Yeah, this makes sense. We don't want base.StringPtr here.